src/styles/blocks/card.scss

```diff
@@ -0,0 +1,76 @@
+body {
+  margin: 0;
+  background-color: gray;
+}
+
+.card {
+  margin: 20px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  width: 200px;
+  height: 408px;
+  padding: 32px 19px 16px 19px;
+  flex-direction: column;
+  display: flex;
+  background-color: white;
+
+  &_image {
+    height: 134px;
+    margin-bottom: 40px;
+  }
+
+  &_product-name {
+    font-family: $bodyFont;
+    font-weight: 500;
+    font-size: 12px;
+    color: #060b35;
+    margin: 0;
+    margin-bottom: 4px;
+    line-height: 18px;
+  }
+
+  &_product-code {
+    font-family: $bodyFont;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+    margin: 0;
+  }
+
+  &_price-conteiner {
+    display: flex;
+    align-items: center;
+  }
+
+  &_price {
+    font-family: $bodyFont;
+    font-weight: 400;
+    font-size: 12px;
+    color: #616070;
+    padding: 0;
+    margin: 0;
+    margin-right: 85px;
```

Comment:
Using a fixed `margin-right` value to separate elements can be inflexible. Consider using `justify-content` on the `.card_price-conteiner` to handle spacing between the price label and the price number more responsively.